src/v/kafka/server/errors.h

```diff
@@ -47,6 +47,8 @@ constexpr error_code map_topic_error_code(cluster::errc code) {
         return error_code::invalid_request;
     case cluster::errc::throttling_quota_exceeded:
         return error_code::throttling_quota_exceeded;
+    case cluster::errc::no_update_in_progress:
```

Comment:
Nice commit comment, however it seems as though each bullet point is a seperate individual bug fix. As such they should be separate commits, would make it easier to review as well.

Reply:
agree